app/models/mfa_config.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Multi Factor Authentication (MFA) configuration for a user.
+class MfaConfig < ActiveRecord::Base
+  belongs_to :user
```

Comment:
To your point about how this goes against the devise convention, is there a way for us to reference `resource` or the generic model that devise can be used for?

Reply:
Hmmm, maybe. Maybe we could have a polymorphic reference from MfaConfig to whatever the generic table is

Reply:
It's been a month but I think the main reason I avoided making this polymorphic was to keep compatibility with how it's done in Piggybank.